src/thunderbots/shared/test/util.cpp

```diff
@@ -0,0 +1,26 @@
+#include <gtest/gtest.h>
+#include <limits>
+extern "C" {
+#include "../constants.h"
+#include "../util.h"
+}
+
+constexpr double EPS = 1e-6;
+
+
+TEST(SharedUtilTest, test_global_to_local_coords)
```

Comment:
Thanks for adding unit tests! A few more test cases would be great.

Reply:
done